src/gui/folder.cpp

```diff
@@ -672,7 +672,7 @@ void Folder::slotWatchedPathsChanged(const QSet<QString> &paths, ChangeReason re
                 Q_ASSERT([&] {
                     Q_ASSERT(record.isValid());
                     // we don't intend to burn to many cpu cycles so limit this check on small files
-                    if (!record.isVirtualFile() && record._fileSize < 1_mb) {
+                    if (!record.isVirtualFile() && record._fileSize < static_cast<qint64>(1_mb)) {
```

Comment:
Wouldn't it make sense to cast the `_fileSize` property or make it the right type right away? (Just a small nitpick, this is fine as-is, too).